pkg/controller/kubedirectorcluster/cluster.go

```diff
@@ -206,9 +206,57 @@ func (r *ReconcileKubeDirectorCluster) syncCluster(
 				shared.EventReasonCluster,
 				"stable",
 			)
+
+			amIBeingConnectedToThis := func(otherCluster kdv1.KubeDirectorCluster) bool {
+				for _, connectedName := range otherCluster.Spec.Connections.Clusters {
+					if cr.Name == connectedName {
+						return true
+					}
+				}
+				return false
+			}
+
+			// Once the cluster is deemed ready, if this cluster is connected to any cluster then
+			// we need to notify that cluster that configmeta here has
+			// changed, so bump up configMetaGenerator for that cluster
+			allClusters := &kdv1.KubeDirectorClusterList{}
+			shared.List(context.TODO(), allClusters)
+			// notify clusters to which this cluster is
+			// connected
+			for _, kubecluster := range allClusters.Items {
+				if amIBeingConnectedToThis(kubecluster) {
+					shared.LogInfof(
+						reqLogger,
+						cr,
+						shared.EventReasonConfigMap,
+						"cluster {%s} is connected to another cluster {%s} updating its configmeta",
+						cr.Name,
+						kubecluster.Name,
+					)
```

Comment:
This log message is a little ambiguous/misleading about the direction of the connection. As far as our representation goes, it's kubecluster.Name that "is connected to" cr.Name.

Also, it seems like we would want to log this event on the kubecluster object, not on the cr object?

Reply:
Added